### ISSUE TITLE
Android app won't launch after building adaptive icons

### ISSUE THREAD

**Issue by codegrue** [fluttercommunity/flutter_launcher_icons#162]
The foreground and background png files are there and everything looks good. The icon looks correct as designed once installed. However, the app won't launch. It just closes as soon as clicked. Removing the adaptive icons and redeploying restores the ability to launch, so I know it's related to the adaptive artwork. Any ideas why?

**Comment by kevincherryholme** [comment]
I am experiencing this on two flutter apps as well. Android adaptive icons prevent the app from launching. @MarkOSullivan94  Any idea why this could be?

**Comment by codegrue** [comment]
I was using just a foreground image. I ended up making two images one for foreground and one for background. Background has no alpha channel. Foreground does, and has transparency as recommended. This seemed to have fixed the issue.

**Comment by kevincherryholme** [comment]
@codegrue Thanks for the quick reply! Do you mind sharing your configuration for your icons? I want to make sure that is correct while I make sure there is no Alpha channel on my background image. Thank you!

**Comment by kevincherryholme** [comment]
I found it!!!! my **launch_background.xml** in android/app/src/main/res/drawable was misconfigured.

I had
```
<?xml version="1.0" encoding="utf-8"?>
<!-- Modify this file to customize your launch splash screen -->
<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:drawable="@android:color/white" />

    <!-- You can insert your own image assets here -->
    <item>
        <bitmap
            android:gravity="center"
            android:src="@mipmap/ic_launcher" />
    </item>
</layer-list>
```

And switching to the default
```
<?xml version="1.0" encoding="utf-8"?>
<!-- Modify this file to customize your launch splash screen -->
<layer-list xmlns:android="http://schemas.android.com/apk/res/android">
    <item android:drawable="@android:color/white" />

    <!-- You can insert your own image assets here -->
    <!-- <item>
        <bitmap
            android:gravity="center"
            android:src="@mipmap/launch_image" />
    </item> -->
</layer-list>
```

fixed the issue. My icons are correct and the app launches successfully.
Notice <item> was NOT commented out!

**Comment by codegrue** [comment]
Do you have files at "@mipmap/ic_launcher"? The <item> section is for showing an image on the splash screen, not the app's icon. Maybe it was referencing a file that wasn't there...

**Comment by kevincherryholme** [comment]
Yeah exactly, its a completely different feature, and my guess was that the resource was missing too, but it's not, the resource is there. What I did was I created another resource but with a different name than ic_launcher, and referenced that isntead, and it worked all around perfectly.

I have no idea why referencing "ic_launcher" for the splash screen makes the app crash when I use adaptive icons.
